pycbc/events/stat.py

```diff
@@ -599,12 +603,75 @@ def reassign_rate(self, ifo):
 
     def coinc_multiifo(self, s, slide,
                        step, **kwargs): # pylint:disable=unused-argument
-        """Calculate the final coinc ranking statistic"""
-        ln_coinc_rate = coinc_rate.combination_noise_coinc_lograte(
+        # ranking statistic is -ln(expected rate density of noise triggers)
+        # plus normalization constant
+        ln_noise_rate = coinc_rate.combination_noise_lograte(
                                   s, kwargs['time_addition'])
         loglr = - ln_coinc_rate + self.benchmark_lograte
         return loglr
 
+class ExpFitSGFgBgRateStatistic(PhaseTDStatistic, ExpFitSGBgRateStatistic):
+    def __init__(self, files):
+        # read in background fit info and store it, also use newsnr_sgveto
+        ExpFitSGBgRateStatistic.__init__(self, files)
+        # Use PhaseTD statistic single.dtype
+        PhaseTDStatistic.__init__(self, files)
+        self.single_dtype.append(('benchmark_sigma', numpy.float32))
+        self.benchmark_sigma = 3.0 * numpy.log(numpy.amin([self.fits_by_tid[ifo]['median_sigma']
+                                   for ifo in ['H1', 'L1']], axis=0))
```

Comment:
this needs to be unpacked a bit (split into 2 lines, eg the first one obtains the sigmas we want, second one converts them into log of estimated network sensitive volume) and commented
also, I would name it something like 'benchmark_logvol' since it's the log of a sensitive volume